src/sentry/api/bases/organization.py

```diff
@@ -315,7 +327,7 @@ def convert_args(self, request, organization_slug, *args, **kwargs):
 class OrganizationReleasesBaseEndpoint(OrganizationEndpoint):
     permission_classes = (OrganizationReleasePermission,)
 
-    def get_projects(self, request, organization):
+    def get_projects(self, request, organization, project_ids=None):
```

Comment:
Should this method use this parameter? Or are you adding it to handle spreads elsewhere?